ui/src/components/app-form/app-form.tsx

```diff
@@ -181,33 +185,24 @@ export const AppForm = ({ id }: AppFormProps): React.ReactElement => {
     );
   };
 
-  function setFocus(focus: boolean): void {
+  function handleFocus(focus: boolean): void {
     setIsFocused(focus);
   }
 
-  const onFormSubmit: SubmitHandler<AppFormInput> = ({
-    display_name,
-    description,
-    framework,
-    thumbnail,
-    filepath,
-    conda_env,
-    custom_command,
-    profile,
-  }) => {
-    const displayName = getFriendlyDisplayName(display_name);
+  const onFormSubmit: SubmitHandler<AppFormInput> = (data) => {
+    const displayName = getFriendlyDisplayName(data.display_name);
     if (profiles && profiles.length > 0) {
       const payload: AppFormInput = {
         jhub_app: true,
         display_name: displayName,
-        description,
-        framework,
-        thumbnail,
-        filepath,
-        conda_env,
+        description: data.description,
+        framework: data.framework,
+        thumbnail: data.thumbnail,
+        filepath: data.filepath,
+        conda_env: data.conda_env,
         env: getFriendlyEnvironmentVariables(variables),
-        custom_command,
-        profile,
+        custom_command: data.custom_command,
+        profile: data.profile,
```

Comment:
Why were these changed to using the data object?

Reply:
My AI assistant suggested that I use data, so I did. Reverted back to original

<img width="497" alt="image" src="https://github.com/nebari-dev/jhub-apps/assets/885376/4a68e117-e83b-4433-b80c-52452d4e3c83">
<img width="497" alt="image" src="https://github.com/nebari-dev/jhub-apps/assets/885376/4a68e117-e83b-4433-b80c-52452d4e3c83">